src/app/views/App.tsx

```diff
@@ -80,6 +81,7 @@ class App extends Component<IAppProps, IAppState> {
   private currentTheme: ITheme = getTheme();
   private statusAreaMobileStyle = appStyles(this.currentTheme).statusAreaMobileScreen;
   private statusAreaLaptopStyle = appStyles(this.currentTheme).statusAreaLaptopScreen;
+  private displayNewFeatureCoachmark = localStorage.getItem('tour');
```

Comment:
It would make sense to abstract this implementation to a function that returns a boolean. 
```suggestion
  private displayNewFeatureCoachmark = shouldDisplayNewFeatureCoachMark();
```

Subsequently it would change the check to something truthy.

`{this.displayNewFeatureCoachmark && <NewFeature />}`

or even 
`{shouldDisplayNewFeatureCoachMark() && <NewFeature />}`